CHAID/CHAID.py

```diff
@@ -133,6 +137,12 @@ def __init__(self, choices=None, split=None, indices=None, node_id=0, parent=Non
         self._members = None
         self.is_terminal = is_terminal
 
+        metadata = dep_v.metadata
+        poss_members = [
+            metadata[k] for k in metadata.keys() if k != -1.0
```

Comment:
This possible members is only used in the case that there are 0 respondents that answered that way. In the case that there are 'N/A' answers, they are included in the members list but only on the nodes they are actually in.